src/generic_ui/polymer/cloud-install.html

```diff
@@ -32,20 +29,26 @@
     }
     uproxy-button {
       background-color: #34AFCE;  /* blue */
+      margin: 4px 0;
       padding: 0.5em;
       width: 100%;
       font-size: 14px;
+      text-transform: none;
```

Comment:
Meant to say (thanks for flagging!), I believe @ohnocassio's intention is to take away `text-transform: uppercase` across the board, but wasn't sure if it'd be better to make that transition gradual, or to have this PR change all of them. Now that you mention it, seems like we might as well change them all now, so I'll give that a shot.